packages/styles/scss/patterns/blocks/contentgroupcards/_contentgroupcards.scss

```diff
@@ -40,12 +38,10 @@
     }
 
     .#{$prefix}--content-group {
-      @include carbon--breakpoint(sm) {
-        @include carbon--make-col(4, 4);
-        .#{$prefix}--content-group-cards__row {
-          padding-left: $carbon--layout-01;
-          padding-right: $carbon--layout-01;
-        }
+      @include carbon--make-col(4, 4);
```

Comment:
Doesn't this just set 100%? Do we need that when a block element by default is 100%?

Reply:
Wondering if this got missed.

Reply:
I've removed it in the latest commit, thanks!